packages/quiz/src/hooks/use-question.ts

```diff
@@ -33,12 +37,15 @@ export default function useQuestion({
   config,
   currentAnswer,
   syntaxHighlighterTheme,
-  questionBodyRenderer,
+  handleSubmitAnswer = defaultSubmitAnswerHandler,
```

Comment:
needed to be able to override this to do what I wanted without breaking existing use